static/js/navbar_alerts.js

```diff
@@ -169,8 +190,14 @@ export function initialize() {
     });
 }
 
-export function open($process) {
-    $("[data-process]").hide();
-    $process.show();
+export function open(args, rendered_alert_content_html) {
+    const rendered_alert_wrapper_html = $(render_navbar_alert_wrapper(args));
+
+    rendered_alert_wrapper_html.first().prepend(rendered_alert_content_html);
```

Comment:
Why do you have this first/prepend logic here?  I don't understand why it's necessary.

Reply:
Because `navbar_alert_wrapper.hbs` has 2 divs first is a wrapper div and the second is for bankruptcy alert. 
If we didn't use `.first()` it will also add the alet content in bankruptcy div.